src/modules/schedule/containers/professional-schedule-form/professional-schedule-form.tsx

```diff
@@ -0,0 +1,144 @@
+import { type FC, Fragment, useState } from 'react';
+// components
+import { Divider } from '@/modules/core/components';
+import { DayScheduleSelect } from '@/modules/schedule/components';
+// utils
+import { formatTimeRange, Time } from '@/modules/core/utils/time.utils';
+
+import type { ProfessionalScheduleFormProps } from './professional-schedule-form.interface';
+import styles from './professional-schedule-form.module.scss';
+
+const days = [
+  'monday',
+  'tuesday',
+  'wednesday',
+  'thursday',
+  'friday',
+  'saturday',
+  'sunday',
+] as const;
+
+// const timezoneOffset = 3;
```

Comment:
Will we need this later?

Reply:
Yeah... It's just a draft example. Can we discuss it when we work on api/logic for this step?